flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/JsonFunctionsTest.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.expressions
+
+import org.apache.flink.table.api.ValidationException
+import org.apache.flink.table.planner.expressions.utils.ScalarTypesTestBase
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class JsonFunctionsTest extends ScalarTypesTestBase {
```

Comment:
Do not extend `ScalarTypesTestBase`, let's define testing data for `JsonFunctionsTest` itself. I guess most of data in `ScalarTypesTestBase` is not suitable in `JsonFunctionsTest.

Reply:
OK. Make sense.

Reply:
updated.